facelist.go

```diff
@@ -95,8 +95,9 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 
 		for _,u := range userlist {
 			tempU, _ := graphClient.GetUser(u.ID)
-			u.AboutMe.Value = tempU.AboutMe.Value
-			userlist2 = append(userlist2, u)
+			tempU.AboutMe.Value = html2text.HTML2Text(tempU.AboutMe.Value)
+			tempU.AboutMe.Value = strings.SplitAfterN(tempU.AboutMe.Value, "\n", 2)[0]
```

Comment:
De här kan skrivas som en operation - du får avgöra vad som är mest lättläst :) 

```suggestion
			tempU.AboutMe.Value = strings.SplitAfterN(html2text.HTML2Text(tempU.AboutMe.Value), "\n", 2)[0]
```